content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md

```diff
@@ -7,6 +7,17 @@ menu:
     name: Release notes
 ---
 
+## v1.6.4 [2022-03-15]
+
+### Features
+
+- Add `SASL` support to `Kafka` alerts.
```

Comment:
should mention `SASL` support in the [Kafka AlertNode doc](https://docs.influxdata.com/kapacitor/v1.6/nodes/alert_node/#kafka) or the [Kafka event handlers doc](https://docs.influxdata.com/kapacitor/v1.6/event_handlers/)?

and would this look different in an alert/config example?

Reply:
I am deferring to @docmerlin for answers :-)

Reply:
@kelseiv @lwandzura  definitely in the kafka event handlers doc, that is where the detailed docs should go.